NDTensors/src/lib/Sectors/src/abstractcategory.jl

```diff
@@ -32,14 +19,100 @@ function Base.show(io::IO, cs::Vector{<:AbstractCategory})
   return nothing
 end
 
+Base.isless(c1::AbstractCategory, c2::AbstractCategory) = isless(label(c1), label(c2))
+
+# =================  Misc  ======================
 function trivial(category_type::Type{<:AbstractCategory})
   return error("`trivial` not defined for type $(category_type).")
 end
 
 istrivial(c::AbstractCategory) = (c == trivial(typeof(c)))
 
-function dual(category_type::Type{<:AbstractCategory})
+# name conflict with LabelledNumber.label. TBD is that an issue?
+label(c::AbstractCategory) = error("method `label` not defined for type $(typeof(c))")
```

Comment:
Either we can separate `Sectors.label` and `LabelledNumber.label` functions, or come up with a different name for the label of a category. We could rename this one to `category_label` to be more specific.

Reply:
It's too bad, because label is pretty much the standard term and it is short and sweet. (E.g. I checked the von Delft paper on SU(N) and they frequently use the term "label".) The only other names I've seen are in the category / topology / stat mech literature where they call the labels variously "simple objects", "anyon types", or "anyon sectors" and names like that. But of course those names are never used in the group theory literature.

Reply:
We could have both `group_label` and `category_label`, or not overload `label` from `LabelledNumbers` and have a private `Sectors.label` function.

Having groups define `group_label` and categories define `category_label` seems ok to me, since I doubt that there will be some generic code that accepts either a group or a category and makes use of the label.

Reply:
I would rather avoid having both `group_label` and `category_label`, as they would play the same role and lead to code dupplicate. The name should stay internal and not exposed, so I think `cagerory_label` is the best solution.